src/components/Clients/ClientListTableRow.jsx

```diff
@@ -1,55 +1,43 @@
 // React Imports
-import React, { useState, useContext } from 'react';
+import React, { useContext, useState } from 'react';
 import { Link } from 'react-router-dom';
 // Material UI Imports
 import { useTheme } from '@mui/material/styles';
-import Checkbox from '@mui/material/Checkbox';
+import Button from '@mui/material/Button';
 import IconButton from '@mui/material/IconButton';
 import DeleteOutlineOutlinedIcon from '@mui/icons-material/DeleteOutlineOutlined';
 import PushPinIcon from '@mui/icons-material/PushPin';
 import PushPinOutlinedIcon from '@mui/icons-material/PushPinOutlined';
-// Utility Imports
-import { runNotification } from '../../utils';
 // Context Imports
-import { SelectedUserContext } from '../../contexts';
 import { StyledTableCell, StyledTableRow } from '../Table/TableStyles';
+import { SelectedUserContext } from '../../contexts';
+
+/**
+ * @typedef {import("../../typedefs.js").clientListTableRowProps} clientListTableRowProps
+ */
 
 /**
  * ClientListTableRow Component - Component that generates the individual table
  * rows of clients from data within ClientList
  *
  * @memberof Clients
  * @name ClientListTableRow
+ * @param {clientListTableRowProps} Props - Props for ClientListTableRow
+ * @returns {React.JSX.Element} The ClientListTableRow Component
  */
-
-// determine what gets rendered in the table body
 const ClientListTableRow = ({
```

Comment:
Now that I think about it, we can actually reduce this client table by one more column if we just make the client name cell a clickable hyperlink to their profile. Then we can remove the `client profile` column

Reply:
I can do that. I'll which the Link to Client instead.